### ISSUE TITLE
How to change PG Schema

### ISSUE THREAD

**Issue by GongSong** [maiha/pg-copy-ch#4]
The default schema is public ,
I want to change the schema to the others,
How can I do it?

**Comment by maiha** [comment]
Hi, @GongSong 
I'm not familiar with custom schema and I'm not sure what the problem is, so please elaborate a bit more.

`pg-copy-ch` is executed in the following three steps.
1. fetch PG table names by SQL written in
  https://github.com/maiha/pg-copy-ch/blob/master/src/data/pg/meta.cr#L78-L96
2. fetch PG data as csv
3. import csv into CH

Where is the problem occurring? 1 or 2?
And, do you know how to correct it?

**Comment by GongSong** [comment]
Hi, @maiha 
Thank you for your reply, 
I'm very sorry, my English is poor.

I think, some problem in the step 1 .
I have several schema in my  database, 
but the useful data is not in the "public" schema,
It's in the others schema,  (e.g.): "my_schema".
If excute SQL: “SET search_path to my_schema” before step 1, 
Perhaps the default schema can be changed to the "my_schema".

**Comment by maiha** [comment]
> If excute SQL: “SET search_path to my_schema” before step 1,

Thanks for the nice information. 💯 
I'll try it later. 👍

**Comment by maiha** [comment]
@GongSong 
Released v0.6.2. Could you try it?
* config: Added `[postgres] before_sql` setting

This value will be inserted directly as a string before the query is executed.
Thus, you can specify here, for example, `search_path`.

### without `before_sql`

will execute 

```sql
Copy (...
```

### with `before_sql`

```toml
[postgres]
before_sql = "SET search_path to myschema,public;"
```

will execute 
```sql
SET search_path to myschema,public;
Copy (...
```

**Comment by GongSong** [comment]
Thank you for your good job.

The problem solved.

Please close the issues.
